### ISSUE TITLE
Exposing more attributes on ConnectionHolderBuilder

### ISSUE THREAD

**Issue by dbousamra** [relayr/rabbitmq-scala-client#2]
It'd be nice if I could set the username and password manually, rather than having to form an AMQP connection string. Perhaps the builder could take in the Java clients builder as a starting point.

**Comment by platy** [comment]
True, it would also make it easier if someone is already using the java client. I'm planning to do some work on the client next week so maybe I could do this at the same time.

**Comment by dbousamra** [comment]
Great.

I am happy to take a look and make some contributions also.

**Comment by platy** [comment]
Thanks, they would be welcome
